packages/dds/tree/src/feature-libraries/defaultChangeFamily.ts

```diff
@@ -0,0 +1,177 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ChangeEncoder, ChangeFamily, ProgressiveEditBuilder } from "../change-family";
+import { ITreeCursor } from "../forest";
+import { ChangeRebaser } from "../rebase";
+import { FieldKindIdentifier } from "../schema-stored";
+import { brand } from "../util";
+import { AnchorSet, Delta, FieldKey, ITreeCursorSynchronous, UpPath, Value } from "../tree";
+import {
+    FieldKind,
+    ModularChangeFamily,
+    ModularEditBuilder,
+    FieldChangeset,
+    FieldChangeMap,
+} from "./modular-schema";
+import { forbidden, optional, sequence, value as valueFieldKind } from "./defaultFieldKinds";
+
+export type DefaultChangeset = FieldChangeMap;
+
+/**
+ * Implementation of {@link ChangeFamily} based on the default set of supported field kinds.
+ *
+ * @sealed
+ */
+export class DefaultChangeFamily implements ChangeFamily<DefaultEditBuilder, DefaultChangeset> {
```

Comment:
Can we export a static instance instead of this class?

Reply:
Unortunately, doing so leads to a cycle in the import graph and makes tests (and test discovery) crash.
I'm open to looking more into it with someone that has more experience tackling those issues, but since I don't, and since the static instance is not required, I ended up choosing not to poke that bear.

Reply:
Update: I got confused. This is now done.